src/Microsoft.DotNet.XHarness.CLI/Commands/Android/AndroidGetDeviceCommand.cs

```diff
@@ -42,6 +42,8 @@ protected override Task<ExitCode> InvokeInternal(ILogger logger)
                 return Task.FromResult(ExitCode.PACKAGE_NOT_FOUND);
             }
 
+            _arguments.Verbosity = LogLevel.Critical;
```

Comment:
Please add a comment explaining this

Reply:
I will add after I fixing it, currently it's not working

Reply:
removed